tigerpath/static/app/js/tutorial.js

```diff
@@ -26,17 +28,19 @@ function startIntro() {
         If a course can only satisfy one of multiple requirements, it will show up underneath each of those requirements and be grayed out.
         You can click on a grayed out course to make it count for that requirement.
         <br><br>
-        You can also hover over a requirement to read more information about it, and to search for courses that satisfy that requirement!`
+        You can also hover over a requirement to read more information about it, and to search for courses that satisfy that requirement!`,
       },
       {
-        element: '#netid-btn',
-        intro: "If you would like to change your major, add a nickname, or modify other settings, you can click here and then on \"Settings\"."
+        element: '#main-view-tabs',
+        intro:
+          'If you have AP credits, summer courses, or other external credits you want to show up in your requirements, you can switch tabs here and add them.',
       },
```

Comment:
Could you maybe add step for the print button?
It would say something like along the lines of "Once you are happy with the courses you've added, you can print or export your schedule here to share with your advisors."